website/vercel.json

```diff
@@ -4121,81 +3305,6 @@
       "source": "/best-practices/how-we-structure/5-semantic-layer-marts",
       "destination": "/best-practices/how-we-build-our-metrics/semantic-layer-7-semantic-structure",
       "permanent": true
-    },
-    {
```

Comment:
i think this totally makes sense but not sure if these redirects are something you'd want to double check with the frontend team since they moved the terms page to the blogs/www site. but in terms of data, there was good traction until the mov
![Screenshot 2025-04-10 at 10 52 28](https://github.com/user-attachments/assets/e986f846-800f-42bd-9752-3d510e8fa1a6)
e was done so maybe it's ok?